### DISCUSSION TITLE
Cant connect portainer to Heimdall. cURL error 60: SSL certificate problem: self signed certificate

### DISCUSSION THREAD

**Discussion by ora2j** [linuxserver/Heimdall#867] [App issues]
I want to connect Portainer to Heimdall but every time I want to log in I have this error (cURL error 60: SSL certificate problem: self-signed certificate). Does Heimdall cant use self-signed certificates, is there a way to fix that without buying a certificate. I tried using my proxy to get the cert but Heimdall won't reach Portainer when using the proxy. 

Anybody knows a way to solve this ??
![Capture](https://user-images.githubusercontent.com/68998445/175301648-9b14eded-e805-4fa9-bf92-72c686310b76.PNG)

**Comment by JTMWi** [comment]
Bump

**Comment by keriati** [comment]
Right now the implementation of skipping TLS verification is up to the Enhanced Apps. The Enhanced App for Plex has this implemented for example.
